app/renderer/ui/components/transaction/confirmed/index.tsx

```diff
@@ -46,7 +46,7 @@ export default class ConfirmedTransaction extends React.Component<ConfirmedTrans
         </p>
         <div>
           <p className={styles.date}>{this.props.date}</p>
-          <p className={styles.block}>Confirmed in block #{this.props.block}</p>
+          <p className={styles.block}>Confirmed in epoch #{this.props.block}</p>
```

Comment:
I would leave the `block` text. @aesedepece told us that block height and epoch are the same thing and in the sketch it appears as `block`.